packages/terra-clinical-application-site/src/index.scss

```diff
@@ -16,6 +16,7 @@ body {
   height: 100%;
 }
 
-.terraClinical-ItemView-accessory {
+[class*="ItemView__accessory"],
+[class*="ItemView__accessory"] > * {
```

Comment:
This is really brittle css, but I understand it's just a conversion. It might be worth investigating if the max-width constraint of the accessory should be changed in the ItemView itself. It feels awkward that we can't use it without custom CSS.